Sources/Rendering/OpenGL/ImageMapper/index.js

```diff
@@ -64,10 +64,6 @@ function vtkOpenGLImageMapper(publicAPI, model) {
       model.openGLCamera = model._openGLRenderer.getViewNodeFor(
         ren.getActiveCamera()
       );
-      // is slice set by the camera
-      if (model.renderable.getSliceAtFocalPoint()) {
-        model.renderable.setSliceFromCamera(ren.getActiveCamera());
-      }
```

Comment:
I have removed/disabled this as it feels like an interaction setup, and shouldn't be part of a mapper. Ideally a backend-mapper shouldn't modify user/app facing properties of a core mapper like this. Such events can be easily setup using application side callbacks, etc.

Reply:
Agreed. This functionality should be re-added somewhere else then, which means having to install a listener on the camera in the core mapper.

Reply:
Ideally, I was thinking the application should hook up the camera update events with the call to update the slice. Otherwise, the mapper is too tightly coupled with interaction code.

But if you want to retain the previous functionality (i.e. the `SliceAtFocalPoint` flag), we could do something like this to force the caller to provide the camera which will be tracked:
```js
  publicAPI.setSliceAtFocalPoint = (flagValue, camera) => {
    model.sliceAtFocalPoint = _superSetSliceAtFocalPoint(flagValue);
    if(flagValue)
        camera.onModified(() => setSliceFromCamera(camera));
  };
```

Reply:
Ok I got an implementation working, I will push it to the branch so you can re-review it.